hawaii-international-conference-on-system-sciences-proceedings.csl

```diff
@@ -36,10 +44,10 @@
   <macro name="title">
     <choose>
       <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
-        <text variable="title" font-style="italic" quotes="false"/>
+        <text variable="title" quotes="false" font-style="italic"/>
```

Comment:
why the italics? Not seeing them in the styleguide.

Reply:
same as above